python/tk_framework_desktopserver/settings.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import ConfigParser
+
+from . import logger
+
+logger = logger.get_logger("settings")
+
+
+class Settings(object):
```

Comment:
This class will be the basis of a refactor in tk-framework-desktopstartup to merge the settings from the desktop server and the desktop.